django/contrib/staticfiles/storage.py

```diff
@@ -146,7 +158,14 @@ def url(self, name, force=False):
 
         return unquote(final_url)
 
-    def url_converter(self, name, template=None):
+    def url(self, name, force=False):
+        """
+        Returns the real URL in DEBUG mode.
```

Comment:
"real URL" was already in the existing code (and is used above) so it was just copied and pasted. It refers to the unhashed URL I believe, so "www.foo.com/bar.jpg" vs "www.foo.com/bar.a545de.jpg"